src/wymeditor/editor/trident-pre-7.js

```diff
@@ -115,25 +115,6 @@ WYMeditor.WymClassTridentPre7.prototype._saveCaret = function () {
     wym._doc.caretPos = nativeSelection.createRange();
 };
 
-WYMeditor.WymClassTridentPre7.prototype.insert = function (html) {
-    var wym = this,
-        // Get the current selection
-        range = wym._doc.selection.createRange(),
-        $selectionParents;
-
-    // Check if the current selection is inside the editor
-    $selectionParents = jQuery(range.parentElement()).parents();
-    if ($selectionParents.is(wym._options.iframeBodySelector)) {
-        try {
-            // Overwrite selection with provided html
-            range.pasteHTML(html);
-        } catch (e) {}
-    } else {
-        // Fall back to the internal paste function if there's no selection
-        wym.paste(html);
-    }
-};
-
```

Comment:
The general `insert` works just as well in IE8 through 10 (11 is not this file).